### PR TITLE
Sprint 6/issue 48#reservation screen

### PR DESCRIPTION
**Pull request by DevLuis123**
redesign of the reservation screen, and edit reseration

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -247,3 +247,5 @@ label {
     max-width: 1140px;
   }
 }
+
+}
```
Comment (MauricioNC) [inline app/assets/stylesheets/application.css:251]: add an empty line at the end of your code

File: app/assets/stylesheets/new_reservation.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    background-color: #190482;
+    color: 190482;
+    font-weight: bold;
+    border: none;
+    transition: background-color 0.3s ease-in-out;
+  }
+  
+  form button:hover {
+    background-color: #190482;
+  }
```
Comment (MauricioNC) [inline app/assets/stylesheets/new_reservation.css:58]: add an empty line at the end of your code

File: app/views/reservations/_search_filter_form.html.erb [hunk 3]
```diff
@@ -1,43 +1,41 @@
...
-      <%= form.submit 'Search', class: 'rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium' %>
-      <%= link_to 'Clear', reservations_path, class: 'rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium ml-3 no-underline' %>
-    </div>
   </div>
-<% end %>
+  <div class='flex justify-center p-6'>
+    <%= form.submit 'Search', class: 'mr-6 bg-purple-700 text-white p-2 rounded-lg hover:bg-purple-400' %>
+    <%= link_to 'Clear', reservations_path, class: 'bg-purple-500 text-white p-2 no-underline rounded-lg hover:bg-purple-400' %>
+  </div>
+<% end %>
```
Comment (MauricioNC) [inline app/views/reservations/_search_filter_form.html.erb:41]: add an empty line at the end of your code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by PudinDeveloper — APPROVED** [review]
Try to order the CSS alphabetically and solve the other issues mentioned.

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (DevLuis123) [reply app/assets/stylesheets/application.css:251]: done!

File: app/assets/stylesheets/new_reservation.css (hunk 2 above)
Comment (DevLuis123) [reply app/assets/stylesheets/new_reservation.css:58]: done!

File: app/views/reservations/_search_filter_form.html.erb (hunk 3 above)
Comment (DevLuis123) [reply app/views/reservations/_search_filter_form.html.erb:41]: done!

File: app/assets/stylesheets/new_reservation.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    color: 190482;
+    font-weight: bold;
+    padding: 10px 20px;
+    transition: background-color 0.3s ease-in-out;
+  }
+  
+  form button:hover {
+    background-color: #190482;
+  }
+
```
Comment (MauricioNC) [inline app/assets/stylesheets/new_reservation.css:59]: Add an empty line at the end of your code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by MauricioNC — APPROVED** [review]

**Review by STR1ND3R — APPROVED** [review]

**Pull request merged**